src/main/java/org/myrobotlab/service/Vertx.java

```diff
@@ -27,7 +27,7 @@
  * It currently does not utilize the Vertx event bus - which is pretty much the most important part of Vertx.
  * TODO: take advantage of publishing on the event bus
  * 
- * @see https://medium.com/@pvub/https-medium-com-pvub-vert-x-workers-6a8df9b2b9ee
+ * <a href="https://medium.com/@pvub/https-medium-com-pvub-vert-x-workers-6a8df9b2b9ee">vertx worker</a>

```

Comment:
You can use a link tag inside of a see tag, would probably be best to do that

Reply:
updated

Reply:
errored with attempt reverting